driver-sync/src/main/com/mongodb/client/internal/ClientSessionImpl.java

```diff
@@ -74,8 +83,12 @@ public void commitTransaction() {
         }
         try {
             if (getServerSession().getStatementId() > 0) {
+                ReadConcern readConcern = transactionOptions.getReadConcern();
+                if (readConcern == null) {
```

Comment:
To make SpotBugs happy...even though we've ensured that by the time we get here the read concern can't be null, SpotBugs doesn't know that.